include/micm/configure/solver_config.hpp

```diff
@@ -115,6 +120,13 @@ namespace micm
     // Error string
     std::stringstream last_json_object_;
 
+    // Constructor
+
+    JsonReaderPolicy(RosenbrockSolverParameters parameters)
+        : parameters_(parameters)
+    {
+    }
+
```

Comment:
Does it makes sense to make it `const ref` to avoid copy?
```suggestion
    JsonReaderPolicy(const RosenbrockSolverParameters& parameters)
        : parameters_(parameters)
    {
    }

```